smf/nexus/config.toml

```diff
@@ -39,3 +39,7 @@ mode = "stderr-terminal"
 #mode = "file"
 #path = "logs/server.log"
 #if_exists = "append"
+
+[updates]
+# If not present, accessing the TUF updates repository will fail
+#tuf_trusted_root = "/opt/oxide/nexus/pkg/root.json"
```

Comment:
Where would this `root.json` come from? I'm noticing this part of Nexus config present-but-commented-out everywhere, sorta wondering what the plan is gonna be for getting this running in a dev/test environment.

Reply:
I should probably write up a script for generating one of these, but copying and pasting from `history`:

```
tuftool root init 1.root.json
tuftool root gen-rsa-key 1.root.json ~/test.key -r root snapshot targets timestamp
tuftool root sign -k ~/test.key 1.root.json
```

then creating a repository from the files in `~/in`:

```
tuftool create -k ~/test.key --outdir repo --root 1.root.json \
    --snapshot-expires 'in 3 days' --targets-expires 'in 3 days' --timestamp-expires 'in 3 days' \
    --snapshot-version $(date +%s) --targets-version $(date +%s) --timestamp-version $(date +%s) \
    --add-targets ~/in
```